main.go

```diff
@@ -74,6 +74,12 @@ func init() {
 		PersistentPreRunE: contextualize(handler.CheckVersion, handler.Panic),
 		RunE:              contextualize(handler.Init, handler.Panic),
 	})
+	rootCmd.AddCommand(&cobra.Command{
+		Use:               "disconnect",
+		Short:             "Disconnect from railway",
+		PersistentPreRunE: contextualize(handler.CheckVersion, handler.Panic),
```

Comment:
nit: Not 100% necessary to check if the CLI is up to date here

Reply:
Makes sense, I just stole it from the `init` definition ><